init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/service/InitProgressReporter.java

```diff
@@ -0,0 +1,41 @@
+package uk.ac.ox.ndph.mts.init_service.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+@Service
+public class InitProgressReporter implements AutoCloseable {
+    private BufferedWriter writer;
+    private static final Logger LOGGER = LoggerFactory.getLogger(InitProgressReporter.class);
+
+    @Autowired
+    public InitProgressReporter(@Value("${progress.log.path}") String initLogMountPath) throws IOException {
+        LOGGER.info(String.format("initializing InitProgressReporter with log file [%s]", initLogMountPath));
+        if (StringUtils.hasText(initLogMountPath)) {
```

Comment:
This is related to my previous comment and the location where you can define a default value of empty string

Reply:
In a previous [comment](https://github.com/NDPH-ARTS/mts-services/pull/167#discussion_r594300789), I had a default empty string in the @Value annotation, and you asked if it's required, so I've removed it.
It's either this or remove the empty string in the env settings of docker-compose,
which one do you think we should stick with?